tests/forms_tests/tests/test_renderer.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from django.forms.renderers import (
+    DjangoTemplateRenderer, Jinja2TemplateRenderer, ProjectTemplateRenderer,
+)
+from django.forms.widgets import Input
+from django.test import SimpleTestCase, modify_settings
+from django.utils._os import upath
+
+
+class SharedTests(object):
+
+    def test_installed_apps_template_found(self):
+        """Can find a custom template in INSTALLED_APPS."""
+        renderer = self.renderer()
+        # Found because forms_tests is .
+        tpl = renderer.get_template('forms_tests/custom_widget.html')
+        expected_path = os.path.abspath(
+            os.path.join(
+                upath(os.path.dirname(__file__)),
+                '..',
+                getattr(self, 'expected_widget_dir', 'templates') + '/forms_tests/custom_widget.html',
```

Comment:
Why not just set `expected_widget_dir = 'templates'` as a class attr on `SharedTests` and avoid the need to do this `getattr` twice?